src/main/kotlin/no/skatteetaten/aurora/mokey/model/ApplicationDeployment.kt

```diff
@@ -17,7 +17,9 @@ data class ApplicationDeployment(
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class ApplicationDeploymentSpec(
     val applicationId: String,
+    val applicationName: String?,
     val applicationDeploymentId: String,
+    val applicationDeploymentName: String?,
```

Comment:
Both applicationName and applicationDeploymentName are non-null in class ApplicationDeploymentSpec in boober. 
Why make them nullable here?

Reply:
Ok, I agree.